tests/test_cli.py

```diff
@@ -16,6 +16,17 @@ def test_app_run_is_called(mocked_run):
     assert mocked_run.called
 
 
+@mock.patch('bottery.cli.App.run')
+@mock.patch('bottery.cli.App.stop')
+def test_app_stop_is_called(mocked_run, mocked_stop):
```

Comment:
the order of the parameters here is wrong. Alse, suggesting a name:

`def test_keyboard_interrupt_correctly_close_app(mocked_stop, mocked_run)`